packages/account-v2/src/pages/TwoFactorAuthentication/TwoFactorAuthenticationDisabled.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { ComponentProps, useEffect } from 'react';
+import { twMerge } from 'tailwind-merge';
+import { useAuthorize, useTwoFactorAuthentication } from '@deriv/api-v2';
+import { Loader, Text } from '@deriv-com/ui';
+import { Timeline } from '../../components/Timeline';
+import { TWO_FACTOR_AUTHENTICATION_URLS } from '../../constants';
+import { TwoFactorAuthenticationForm } from '../../containers/TwoFactorAuthenticationForm';
+import { TwoFactorAuthenticationArticle } from './TwoFactorAuthenticationArticle';
+import { TwoFactorAuthenticationQRCode } from './TwoFactorAuthenticationQRCode';
+
+const TwoFactorAuthenticationLink = ({ children, className = '', ...rest }: ComponentProps<'a'>) => {
+    return (
+        <a
+            {...rest}
+            className={twMerge('hover:underline font-bold', className)}
+            rel='noopener noreferrer'
+            target='_blank'
+        >
+            {children}
+        </a>
+    );
+};
+
+export const TwoFactorAuthenticationDisabled = () => {
+    const { data: authorizeData } = useAuthorize();
+    const { email = '' } = authorizeData ?? {};
+    const { data: secretKeyData, isLoading, mutate } = useTwoFactorAuthentication();
+    const { secretKey } = secretKeyData ?? {};
+    useEffect(() => {
+        mutate({ totp_action: 'generate' });
+    }, [mutate]);
+
+    return (
+        <div className='flex flex-1 w-full h-full sm:flex-col-reverse sm:col-span-2'>
+            <section className='py-0 px-[14px]'>
+                <Text
+                    align='start'
+                    as='h2'
+                    className='ml-[12px] mb-[15px] text-[16px] text-system-light-prominent-text leading-normal sm:mr-1.2rem'
```

Comment:
shouldnt this be in px?

Reply:
Can u check the 15px margin bottom from figma?. Also can use size property of `Text` component

Reply:
Alright @likhith-deriv, what value should it be then? Because I got these measurements from `v1` directly